code/modules/reqs/supplypacks.dm

```diff
@@ -1050,7 +1040,7 @@ ENGINEERING
 
 /datum/supply_packs/engineering/sandbags
 	name = "50 empty sandbags"
-	contains = list(/obj/item/stack/sandbags_empty/full)
+	contains = list(/obj/item/stack/sandbags_empty/half)
 	cost = 10
```

Comment:
A) You didnt update description
B) Atomize this

Reply:
Haha atomisation go brrr, will update tomorrow